src/profile/index.js

```diff
@@ -0,0 +1,11 @@
+import profileReducer from './reducers';
+import profileSaga from './sagas';
+import ConnectedProfilePage from './components/ProfilePage';
+import configureProfileApiService from './services';
+
+export {
```

Comment:
So this is cool!  The whole profile module has four things it exports.  That's its entire interface with the outside world.

Reply:
This means that to use the profile module, you just import from this file, rather than importing from `./profile/<anything else>`.  Technically there isn't a good way to enforce that, that I'm aware of, short of moving the profile organism into its own repo, though.

Reply:
Also, the profile module should not be reaching outside of the `profile` directory... ever!  The one exception right now is that it uses things in the 'common' module, but as mentioned above, all of those should be shuffled off into other repos/npm modules, and/or it represents common code that multiple modules may need.  Otherwise, `profile` doesn't reach up into the MFE boilerplate at all now.

Reply:
Mostly?  I'd say if you find yourself looking outside a module for something, think twice.  I kinda fleshed this out in those "principles" in the PR description _after_ I wrote these comments...  Within a module, any time you have some code reuse, that code's likely to be in a "common" sibling directory to the places using it, I'll bet.